### PR TITLE
Fix pip ooniprobe install (upgrade)

### PR DESCRIPTION
**Pull request by anadahz**
The issue is introduced by setuptools v34.0.0
http://setuptools.readthedocs.io/en/latest/history.html#v34-0-0

### CODE REVIEW

File: updater/versions/update-1.py [hunk 1]
```diff
@@ -272,6 +272,13 @@ def _perform_update():
 
     shutil.copyfile("/usr/share/zoneinfo/UTC", "/etc/localtime")
 
+    # Fix pip bug introduced in setuptools v34.0.0
+    # http://setuptools.readthedocs.io/en/latest/history.html#v34-0-0
+    check_call(["apt-get", "-y", "install", "-t", "stretch", "python-pip"])
+    # Remove previously installed python packages
+    check_call(["apt-get", "-y", "autoremove"])
+    check_call(["pip", "install", "--upgrade", "setuptools"])
```
Comment (hellais) [inline updater/versions/update-1.py]: We should explicitly specify the version number of setuptools with `setuptools==KNOWN_WORKING_VERSION` and omit `--upgrade`

**Review by hellais — CHANGES_REQUESTED** [review]
Two things should be done to make this more robust:

1. Specify an explicit setuptools version on the pip install line

2. Run apt-get update before doing the apt-get install (if it's not already being run as part of the update script).

**Comment by anadahz** [comment]
Thanks @hellais made the required changes.
LGTM and sign the hotfix updates?

**Comment by hellais** [comment]
Yes this looks good. I am going to however use this as an opportunity to also update the GPG key to something that doesn't expire in a year.

**Pull request merged**